multiomics/README.md

```diff
@@ -0,0 +1,50 @@
+# Multiomics Datasets
+
+This repository contains a collection of publicly available multiomics datasets that have been slightly modified for use
```

Comment:
In GitHub, a repository refers to `https://github.com/pykale/data` so it will be better not to call a subdirectory as repo too. See https://github.com/pykale/data/tree/main/tabular/cardiac_landmark_uncertainty, https://github.com/pykale/data/tree/main/images/ShefPAH-179 and https://github.com/pykale/data/tree/main/images/office to be consistent with them.

Reply:
Changed wording to avoid confusion.